### DISCUSSION TITLE
Attribute listing.size_meters_squared -- Not working?

### DISCUSSION THREAD

**Discussion by wainaze** [AnthonyBloomer/daftlistings#148] [Q&A]
Hi!
Thanks a milltion for making this available, I'm trying to get the size of the properties I'm searching for:

from daftlistings import Daft, Location, SearchType, PropertyType
daft = Daft()

daft.set_location([Location.ASHTOWN_DUBLIN])
daft.set_property_type(PropertyType.HOUSE)
daft.set_search_type(SearchType.RESIDENTIAL_SALE)

listings = daft.search()

for listing in listings:
    print(listing.price)
    print(listing.size_meters_squared)`

OUTPUT
Search complete. Found 2 listings.
€485,000
Traceback (most recent call last):
  File "daft.py", line 28, in <module>
    print(listing.size_meters_squared)
AttributeError: 'Listing' object has no attribute 'size_meters_squared'    

Am I missing something??

**Comment by sdfordham** [comment]
The PyPI version does not have this yet since its a recent addition, you will have to wait on @AnthonyBloomer to upload a new wheel.

Two options:
* Build from source clone this repo, install `setupttools` and `wheel` and then run `python setup.py bdist_wheel` you will get a folder called dist and a wheel inside that you can install with `pip`.
* Do `print(listing._result["floorArea"]["value"])` probably inside a try block although I imagine most house listings do have this.

**Comment by AnthonyBloomer** [comment]
Published a new version to PyPi which includes the size_meters_squared method. https://pypi.org/project/daftlistings/2.0.2/
